lib/json_api_resource/resource.rb

```diff
@@ -35,6 +35,7 @@ def persisted?
     def save
       run_callbacks :save do
         self.client.save
+        self.meta = self.client.last_request_meta
```

Comment:
the 0_x branch? Eventually, probably. There's a ton of projects that use this, so let's maybe keep it until we have a chance to port the main projects over to 1.0?

What were the issues with `after_save`?